2552 (#19)
mattpocock/ts-error-translator: packages/engine/errors/2552.md

---
original: "TS2552: Cannot find name '{0}'. Did you mean '{1}'?"
excerpt: "You are trying to reference a function or variable which Typescript cannot find within the scope."
---

> This means that you are trying to reference a variable or method which cannot be resolved within the same scope. It is most likely a typo. 

Reviewer: ```suggestion
This means that you are trying to reference a variable or method which I can't find in this scope. It might be a typo.
```

> Double check that you have spelt it correctly. You even have a suggestion from me, letting you know if `{1}` might be what you're looking for.

Reviewer: ```suggestion
Double check that you have spelled it correctly. You might be looking for {1}?
```
